dcctools/config.py

```diff
@@ -136,7 +203,7 @@ def get_clks_raw(self, system, project):
         clks = None
         clk_zip_path = Path(self.config_json["inbox_folder"]) / "{}.zip".format(system)
         with ZipFile(clk_zip_path, mode="r") as clk_zip:
-            with clk_zip.open("output/{}.json".format(project)) as clk_file:
+            with clk_zip.open(self.get_clk_path(system, project)) as clk_file:
```

Comment:
I'm getting an error around this line of code while trying to run `python projects.py`

```shell
Traceback (most recent call last):
  File "/Users/radamson/code/codi/linkage-agent-tools/projects.py", line 120, in <module>
    run_projects(config, args.project)
  File "/Users/radamson/code/codi/linkage-agent-tools/projects.py", line 61, in run_projects
    run_project(c, timestamp, project_name)
  File "/Users/radamson/code/codi/linkage-agent-tools/projects.py", line 87, in run_project
    project.upload_clks(system, c.get_clks_raw(system, project_name))
  File "/Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py", line 206, in get_clks_raw
    with clk_zip.open(self.get_clk_path(system, project)) as clk_file:
  File "/usr/local/Cellar/python@3.9/3.9.14/Frameworks/Python.framework/Versions/3.9/lib/python3.9/zipfile.py", line 1511, in open
    zinfo = self.getinfo(name)
  File "/usr/local/Cellar/python@3.9/3.9.14/Frameworks/Python.framework/Versions/3.9/lib/python3.9/zipfile.py", line 1438, in getinfo
    raise KeyError(
KeyError: 'There is no item named None in the archive'
```

From poking around I see

```shell
> /Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py(189)get_clk_path()
-> target = f"households/{project}" if self.household_match else project
184                 f"{system}_households.zip" if self.household_match else f"{system}.zip"
185             )
186             clk_zip_path = Path(self.config_json["inbox_folder"]) / archive_name
187             import pdb; pdb.set_trace()
188             with ZipFile(clk_zip_path, mode="r") as clk_zip:
189  ->             target = f"households/{project}" if self.household_match else project
190                 for file_name in clk_zip.namelist():
191                     if target in file_name:
192                         if len(file_name) == len(target) + 21:
193                             return file_name
194
(Pdb) n
> /Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py(190)get_clk_path()
-> for file_name in clk_zip.namelist():
(Pdb) clk_zip
<zipfile.ZipFile filename='/Users/radamson/code/codi/inbox/site_a.zip' mode='r'>
(Pdb) clk_zip.namelist()
['output/name-sex-dob-phone.json', 'output/name-sex-dob-zip.json', 'output/name-sex-dob-addr.json', 'output/metadata-20220922T092600.json']
(Pdb) n
> /Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py(191)get_clk_path()
-> if target in file_name:
(Pdb) n
> /Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py(192)get_clk_path()
-> if len(file_name) == len(target) + 21:
(Pdb) n
> /Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py(190)get_clk_path()
-> for file_name in clk_zip.namelist():
(Pdb) n
> /Users/radamson/code/codi/linkage-agent-tools/dcctools/config.py(191)get_clk_path()
-> if target in file_name:
(Pdb) target
'name-sex-dob-phone'
(Pdb) file_name
'output/name-sex-dob-zip.json'
```

It looks like the project name being searched for doesn't match the path in the archive. Are you also experiencing this error? If not, what values for `target` and and `file_name` do you have.

Reply:
Invoking `python projects.py`.

The files I am using were freshly generated from the latest commit on the master branch of data-owner-tools. The archives produced do not include timestamps on the CLK files. Based on the conversation we just had it looks like you are developing against an old set of data that doesn't reflect the current state of artifacts produced by data-owner-tools.

Reply:
New version reflects current state of artifacts; use `--config config-metadata.json` for testing